<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DifficultyLevels</class>
 <widget class="QDialog" name="DifficultyLevels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="quitBtn">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
       </property>
       <property name="text">
        <string>QUIT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backBtn">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
       </property>
       <property name="text">
        <string>BACK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">color: black;          
font-size: 20px;         
font-family: Arial;
font-weight: bold;
text-align: center;</string>
     </property>
     <property name="text">
      <string>LEVELS OF DIFFICULTY</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>200</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>80</number>
     </property>
     <item>
      <widget class="QPushButton" name="easyBtn">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;       </string>
       </property>
       <property name="text">
        <string>EASY</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mediumBtn">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;       </string>
       </property>
       <property name="text">
        <string>MEDIUM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hardBtn">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;       </string>
       </property>
       <property name="text">
        <string>HARD</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>150</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>200</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
